airflow/config_templates/config.yml

```diff
@@ -1373,6 +1373,23 @@
       type: int
       example: ~
       default: "2"
+    - name: task_track_started
+      description: |
+        Celery task will report its status as 'started' when the task is executed by a worker.
+        This is used in Airflow to keep track of the running tasks and if a Scheduler is restarted
+        or run in HA mode, it can adopt the orphan tasks launched by previous SchedulerJob.
+      version_added: 2.0.0
+      type: boolean
+      example: ~
+      default: "True"
+    - name: adopted_task_timeout
+      description: |
+        Time after which Adopted tasks are cleared by CeleryExecutor. This is helpful to clear
```

Comment:
```suggestion
        Time in seconds after which Adopted tasks are cleared by CeleryExecutor. This is helpful to clear
```
Just to clarify

Reply:
https://github.com/apache/airflow/pull/10949/commits/e4f915efb2b1bd4165d33425799f3bbffd88caa3